scripts/bootstrap.sh

```diff
@@ -10,7 +10,7 @@ if [ -z "$GITHUB_USER" ]; then
   GITHUB_USER=timvideos
 fi
 if [ -z "$GITHUB_REPO" ]; then
-  GITHUB_REPO=HDMI2USB-litex-firmware
+  GITHUB_REPO=LiteX-BuildEnv
```

Comment:
I think this should be lowercase?

Reply:
It works fine either way, I used the case you used in the bug, but am indifferent, changed to lower case.